recipe/meta.yaml

```diff
@@ -1,31 +1,37 @@
 {% set name = "pytest-azurepipelines" %}
-{% set version = "0.8.0" %}
+{% set version = "1.0.5" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 944ae2c0790b792d123aa7312fe307bc35214dd26531728923ae5085a1d1feab
+  sha256: fd08f089bca0fad3c2b6fde9dbea69b975492ffaa8cb637cc987e64af8c18b9c
 
 build:
   noarch: python
   number: 0
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  script: "{{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv"
 
 requirements:
   host:
     - python
     - pip
+    - wheel
   run:
     - python
     - pytest
```

Comment:
Shouldn't we add the following dependencies listed [here](https://github.com/tonybaloney/pytest-azurepipelines/blob/master/setup.py#L38)?

Reply:
Yeah, and then remove `importlib_resources` from `test.requires`.
pytest-nunit is needed for command-line option `--nunit-xml`, see https://github.com/pytest-dev/pytest-nunit#command-line-options and https://inspector.pypi.io/project/pytest-azurepipelines/1.0.5/packages/3d/9f/d2ed8e69421cc770c2f807778eb08882a4fd320c8eebf53370320445a368/pytest-azurepipelines-1.0.5.tar.gz/pytest-azurepipelines-1.0.5/pytest_azurepipelines.py#line.57

Reply:
ok thanks! I saw it needed nunit to run the test and importlib_resources in the test code, how do you know it needs that in the host section?

Reply:
It's for the `run` section

Reply:
```
pytest >=5.0.0
pytest-nunit >=1.0.0,<2.0.0
```
@scotthain Could you add pinnings?  See https://inspector.pypi.io/project/pytest-azurepipelines/1.0.5/packages/3d/9f/d2ed8e69421cc770c2f807778eb08882a4fd320c8eebf53370320445a368/pytest-azurepipelines-1.0.5.tar.gz/pytest-azurepipelines-1.0.5/setup.py#line.38